src/NamingStrategy/UnderscoreNamingStrategy/UnderscoreToCamelCaseFilter.php

```diff
@@ -58,7 +71,7 @@ function ($matches) {
             )
             : new PcreReplacement(
                 '#(' . $pregQuotedSeparator . ')'
-                    . '([^\p{Z}\p{Ll}]{1}|[a-zA-Z]{1})#u',
+                . '([^\p{Z}\p{Ll}]{1}|[a-zA-Z]{1})#u',
```

Comment:
Previous indentation made it more clear that the second line was a continuation of the first argument; I'd argue we should keep it as-is.

Reply:
I updated the code.